infra/example-bootstrap-gcp-simple/variables.tf

```diff
@@ -0,0 +1,34 @@
+variable "project_name" {
+  type        = string
+  description = "name to give to GCP project"
+  default     = null
```

Comment:
`null` ? Maybe better without default, so it will be enforced to have a value later

Reply:
it'll use `coalesce()` to take the `project_id` value as the name if left `null`, which I think is the right thing to do. We don't need to force customer to specify a name for the project - but it's nice to give the option.